examples/kubernetes/dynamic_provisioning_s3/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.csi.aws.com
+parameters:
+  subnetId: subnet-0cd89fc1c1bc6ef51
+  securityGroupIds: sg-0afdb18c8af898a78
+  s3ImportPath: s3://dl-benchmark-result
```

Comment:
I just use `s3` as prefix and seems FSx will support different backend later on.